snapcraft/tests/test_plugin_python.py

```diff
@@ -257,7 +269,7 @@ def test_fileset_ignores(self):
         self.assertListEqual(expected_fileset, fileset)
 
     @mock.patch.object(python.PythonPlugin, 'run')
-    def test_build_fixes_python_shebangs(self, run_mock):
+    def test_pull_fixes_python_shebangs(self, run_mock):
```

Comment:
Why this change? The shebang rewrite is happening in build, and this test tests the build.